modules/claims_api/app/models/claims_api/form_526.rb

```diff
@@ -38,23 +40,29 @@ class Form526
     end
 
     def initialize(params = {})
-      @attributes = []
-      params.each do |name, value|
-        @attributes << name.to_sym
-        begin
-          send("#{name}=", value)
-        rescue StandardError
+      sanatized_params = sanatize_fields(params)
```

Comment:
sanatized -> sanitized

Reply:
Fixed, sorry about that it was early in the morning lol.